intTests/test_hash_table/ht_simplified.c

```diff
@@ -0,0 +1,130 @@
+/**
+ * pure C hash table for call path throttling.
+ *
+ * constraints:
+ *   - static pool of memory
+ *   - uint64_t key
+ *   - uint32_t value
+ *
+ * notes:
+ *   - error when pool exhausted : no eviction or reuse policy
+ *   - fixed number of buckets: we don't rebalance if some buckets get long
+ *     entry chains
+ *
+ * sottile2@llnl.gov // June 2023
+ */
```

Comment:
I'm guessing that this code is taken from somewhere else, so please put the copyright information somewhere in this header.

Reply:
Added authors name.